### ISSUE TITLE
Heart symbol

### ISSUE THREAD

**Issue by mudcube** [googlecreativelab/quickdraw-dataset#37]
What do you think about adding a `heart` symbol category to the app? I believe it would be a good candidate for QuickDraw as it can is fairly language agnostic, very recognizable, and can be easily drawn with a single stroke. 

On the same topic, is this the best place to propose new symbols?

**Comment by llamax1** [comment]
Good point - I also think there should be a place to suggest categories, much like Font Awesome.
